README.rst

```diff
@@ -5,7 +5,7 @@ Matrix Client SDK for Python
   :target: https://pypi.python.org/pypi/matrix-client/
   :alt: Latest Version
 
-This is a Matrix client-server SDK for Python 2.x.
+This is a Matrix client-server SDK for Python 3.x.
```

Comment:
I didn't want to make any promises (which is why I put this on the python 3 branch). I can give it a test later.